packages/rpm/polymc.spec

```diff
@@ -123,7 +123,7 @@ desktop-file-validate %{buildroot}%{_datadir}/applications/org.polymc.PolyMC.des
 %{_libdir}/%{name}/*
 %{_datadir}/%{name}/*
 %{_datadir}/metainfo/org.polymc.PolyMC.metainfo.xml
-%{_datadir}/icons/hicolor/scalable/apps/org.polymc.PolyMC.svg
+%{_datadir}/icons/hicolor/scalable/apps/org.polymc.polymC.svg
```

Comment:
The C is still uppercase

Reply:
Fixed (also realized that as the icon file and not the desktop file... oops). Although it seems Github is acting strange now, and not showing this as being fixed (if you view my branch outside of the this PR it is fixed though).